x-pack/plugins/observability_onboarding/public/components/app/custom_logs/wizard/select_logs.tsx

```diff
@@ -0,0 +1,241 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/* eslint-disable @elastic/eui/href-or-on-click */
+
+import React, { PropsWithChildren, MouseEvent } from 'react';
+import {
+  EuiTitle,
+  EuiLink,
+  EuiButton,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiHorizontalRule,
+  EuiSpacer,
+  EuiCard,
+  EuiIcon,
+  EuiIconProps,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import {
+  StepPanel,
+  StepPanelContent,
+  StepPanelFooter,
+} from '../../../shared/step_panel';
+import { useWizard } from '.';
+import { useKibanaNavigation } from '../../../../hooks/use_kibana_navigation';
+
+export function SelectLogs() {
+  const { navigateToKibanaUrl, navigateToAppUrl } = useKibanaNavigation();
+  const { goToStep, getState, setState } = useWizard();
+
+  function onBack() {
+    navigateToKibanaUrl('/app/observabilityOnboarding/customLogs');
+  }
+
+  return (
+    <StepPanel
+      title={i18n.translate(
+        'xpack.observability_onboarding.selectLogs.chooseType',
+        {
+          defaultMessage: 'Choose what logs to collect',
+        }
+      )}
+    >
+      <StepPanelContent>
+        <LogsTypeSection
+          title={i18n.translate(
+            'xpack.observability_onboarding.selectLogs.customLogs',
+            {
+              defaultMessage: 'Custom logs',
+            }
+          )}
+        >
+          <EuiFlexGroup>
+            <EuiFlexItem grow={false} style={{ width: '50%' }}>
+              <OptionCard
+                title={i18n.translate(
+                  'xpack.observability_onboarding.selectLogs.streamLogFiles',
+                  {
+                    defaultMessage: 'Stream log files',
+                  }
+                )}
+                iconType="desktop"
+                onClick={() => {
+                  setState({ ...getState(), logsType: 'log-file' });
+                  goToStep('configureLogs');
+                }}
+                isSelected={false}
```

Comment:
we can probably remove the `isSelected` prop since the cards are used for navigation rather than selection.

Reply:
Addressed in [2826d0f](https://github.com/elastic/kibana/pull/156182/commits/2826d0f297aef033a32f9ea6675aea86ce7f5d48).